### DISCUSSION TITLE
The Activity Bar, Status Bar, Panel, Side Bar, Editor are `surface`, how about the items under them? such as `View`, `Editor Toolbar` ?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [microsoft/vscode-discussions#378] [Extension Development QnA] [answered]
How to understand `surface` in VSCode? 

>[Activity Bar](https://code.visualstudio.com/api/ux-guidelines/overview#activity-bar)
The [Activity Bar](https://code.visualstudio.com/api/ux-guidelines/activity-bar) is a core navigation surface in VS Code. Extensions can contribute items to the Activity Bar that function as [View Containers](https://code.visualstudio.com/api/references/contribution-points#contributes.viewsContainers) that render [Views](https://code.visualstudio.com/api/ux-guidelines/views) in the Primary Sidebar.

I want to know in this image, 
![image](https://user-images.githubusercontent.com/12727582/210912749-7dcdef52-cc6e-4bc3-8ebe-7f6ecfb551bf.png)

The Activity Bar, Status Bar, Panel, Side Bar, Editor are `surface`, how about the items under them? such as `View`, `Editor Toolbar` ?

**Comment by isidorn** [comment, marked as answer]
I think the picture is self explanatory :)
